skaffold.yaml

```diff
@@ -35,9 +39,30 @@ kind: Config
     - *OBDEMO-RCS-UI
     - *OBDEMO-RCS-API
     - *OBDEMO-RS
-
+    local-artifacts: &local-artifacts
```

Comment:
Added `local-artifacts` reference to be used on the profile `obdemo-bank-local` defined below.